sonarqube/tests/test_unit_components.py

```diff
@@ -0,0 +1,382 @@
+import re
+from random import randrange
+
+import mock
+import pytest
+
+from .common import HOST, METRICS, PORT
+
+pytestmark = [pytest.mark.unit]
+
+
+@mock.patch("datadog_checks.sonarqube.check.SonarqubeAPI")
```

Comment:
Please avoid decorator side effects and use `mocker` from https://github.com/pytest-dev/pytest-mock which we already ship

See https://pytest-mock.readthedocs.io/en/latest/remarks.html#why-bother-with-a-plugin

Reply:
Done